pkg/cmd/tknpac/bootstrap/kubestuff.go

```diff
@@ -44,9 +46,24 @@ func checkSecret(ctx context.Context, run *params.Run, opts *bootstrapOpts) bool
 }
 
 // check if we have the namespace created
-func checkNS(ctx context.Context, run *params.Run, opts *bootstrapOpts) (bool, error) {
-	ns, err := run.Clients.Kube.CoreV1().Namespaces().Get(ctx, opts.targetNamespace, metav1.GetOptions{})
-	return ns.GetName() != "", err
+func checkNS(ctx context.Context, run *params.Run, targetNamespace string) (bool, error) {
+	ns, err := run.Clients.Kube.CoreV1().Namespaces().Get(ctx, targetNamespace, metav1.GetOptions{})
+	if err != nil {
+		return ns.GetName() != "", err
```

Comment:
if err occurs `ns.GetName() != ""`  ns might be nil and this would break
the prev thing was correct i think